myuw/static/css/mobile.less

```diff
@@ -828,6 +820,52 @@ label { font-weight: normal; margin-bottom: auto; }
         padding-left: 16px;
         ul { list-style-type: none; padding-left: 0; }
     }
+	// Move the action button out of the .card
+	.card-action-btn { 
+	        margin-bottom: 16px; margin-top: 8px;
+                    &:hover, &:focus {background-color:#e8e3d3;border-color:#e8e3d3;}                   
+                    a {color:#444;
+                    	&:hover, &:focus {color: #000; text-decoration: none; }
+                    }     
+         }
+	//Sub-heading style 
+	.myuw-card-heading {
+		font-size: 16px; 
+		font-family:'Roboto', sans-serif;
+		font-weight: 500;
+		color:#85754d;
+		line-height: 1.25;
+		}
+	
+    // Application cards styles
+   .myuw-card.application-card { //remove top padding
+		    margin-top: 0;
+	    }
+
+    .application-card {
+
+	    .myuw-btn-big {
```

Comment:
`myuw-btn-big`

If we plan to use this style of button elsewhere, we should probably move it to the components.less file and not nest it here under .application-card. As written, these styles only apply for the application-card, but its name suggests it is a widely used style.

Reply:
I'm also a little confused how it relates to `card-action-btn`. You're using them together which suggests the `myuw-btn-big` is sort of an extension of `card-action-btn`, but the naming scheme is different. It would maybe make more sense to use "myuw-btn" and "myuw-btn-big". I would put both in the components.less.

If the `myuw-btn-big` is just a larger `card-action-btn`, I'm wondering why the switch to Roboto typeface?

Reply:
Your are right that .myuw-btn-big is an extension of .card-action-btn. 
The Roboto typeface looks nicer in the button - more refined button for applicant! 
.myuw-btn-big is moved to component.less.

Correcting the .card-action-btn will require more templates changes and not critical for this release, so will add to the backlog.

Reply:
IMO, if we like Roboto for buttons we should just use it for all the buttons, not just the big ones.

When you tackle refactoring the buttons, I'd recommend only building on the `btn` class instead of both `btn` and `btn-default`. You aren't using a lot of the `btn-defalut` styles and I think it'd be better to just make the `myuw-btn` more complete.

"btn btn-default card-action-btn myuw-btn-big